### DISCUSSION TITLE
Collapseable systry

### DISCUSSION THREAD

**Discussion by paridhi089** [qtile/qtile#3471] [Ideas] [answered]
Would it be possible to make a function to hide the systray with the press of a button or more like a collapseable design. because the systray icons ruin the looks.

**Comment by elParaguayo** [comment, marked as answer]
Put it in a `widget.WidgetBox`?

http://docs.qtile.org/en/stable/manual/ref/widgets.html#widgetbox

**Reply by SweepingBishops** [reply]
That's what I do too.  
```
widget.WidgetBox(widgets=[  
                    widget.NetGraph(),  
                    widget.Systray(),  
                    ],  
                close_button_location='right',  
                text_closed='',  
                text_open='',  
                fontsize=16,  
                ),
```

**Comment by PhlegmaTREEc** [comment]
Can I maybe hijack this topic a bit and expand the original question... Would it be possible to toggle the WidgetBox status with keyboard shortcut?

**Reply by elParaguayo** [reply]
Yes!
``` python
lazy.widget["widgetbox"].toggle()
```

**Reply by PhlegmaTREEc** [reply]
OK, that's good to hear :smile: 
I may need some help with implementing it
`Key([mod], "v", lazy.widget["widgetbox"].togle(), desc="Toggle WidgetBox"),`
Right now I just slapped it in may keybindings section, but no luck. I assume I need to change to fit my config, or do some other change elsewhere?

**Reply by elParaguayo** [reply]
`toggle` with 2 `g`s.

**Reply by PhlegmaTREEc** [reply]
F***
I'm really sorry about that. Now it works like a charm. Thank you
